src/pages/test-report/cat-adi-part2/components/controlled-stop/__tests__/controlled-stop.spec.ts

```diff
@@ -0,0 +1,225 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { ControlledStopComponent } from '../controlled-stop';
+import { IonicModule } from 'ionic-angular';
+import { StoreModule, Store } from '@ngrx/store';
+import { testsReducer } from '../../../../../../modules/tests/tests.reducer';
+import { testReportReducer } from '../../../../test-report.reducer';
+import { StoreModel } from '../../../../../../shared/models/store.model';
+import { MockComponent } from 'ng-mocks';
+import { CompetencyButtonComponent } from '../../../../components/competency-button/competency-button';
+import { TickIndicatorComponent } from '../../../../../../components/common/tick-indicator/tick-indicator';
+import {
+  DrivingFaultsBadgeComponent } from '../../../../../../components/common/driving-faults-badge/driving-faults-badge';
+import {
+  SeriousFaultBadgeComponent,
+} from '../../../../../../components/common/serious-fault-badge/serious-fault-badge';
+import {
+  DangerousFaultBadgeComponent,
+} from '../../../../../../components/common/dangerous-fault-badge/dangerous-fault-badge';
+import { StartTest } from '../../../../../../modules/tests/tests.actions';
+// TO-DO ADI Part2: implement correct actions
+import {
+  ControlledStopAddDrivingFault,
+  ControlledStopRemoveFault,
+} from '../../../../../../modules/tests/test-data/cat-b/controlled-stop/controlled-stop.actions';
```

Comment:
Controlled Stop has been made common as part of Home tests , this file no longer exists